core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java

```diff
@@ -407,18 +400,12 @@ private static boolean hasFakeSequence(PoolableElementView view) {
 
   @Override
   public boolean hasLibrariesWithoutIndex() {
-    List<PoolableElementView> views = new ArrayList<>(getPoolableElementViews());
-    for (int i = 0; i < views.size(); i++) {
-      if (views.get(i).getIndices().isEmpty()) {
-        return true;
-      }
-    }
-    return false;
+    return getPoolDilutions().stream().map(pd -> pd.getPoolableElementView()).anyMatch(v -> v.getIndices().isEmpty());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)